src/test/scala/com/mwz/sonar/scala/scalastyle/ScalastyleRulesRepositorySpec.scala

```diff
@@ -46,8 +47,8 @@ class ScalastyleRulesRepositorySpec extends AnyFlatSpec with Matchers with Inspe
   }
 
   it should "include all Scalastyle inspections" in new Ctx {
-    ScalastyleInspections.AllInspections should have size 72 // 31 templates + 41 default rules
-    ScalastyleInspections.AllInspectionsByClass.size shouldBe ScalastyleInspections.AllInspections.size
+    ScalastyleRules.rules.length shouldBe 72 // 31 templates + 41 default rules
+    ScalastyleRules.rules.map(r => r.key -> r).iterator.toMap.size shouldBe ScalastyleRules.rules.length
```

Comment:
Any particular reason for not using `have size` or `have length`?

Also, for the second test, I believe it would be better to ensure that all the keys are unique.

```scala
ScalastyleRules.rules.distinctBy(_.key) shouldBe ScalastyleRules.rules
```

Reply:
> Any particular reason for not using `have size` or `have length`?

Yes, so Chain doesn't have the size property (it has length) and I found the autocompletion to work better in metals when you first access the property and then use matchers for comparing the value. It couldn't easily infer the `have length` syntax, so I settled on `.length shouldBe`.

> Also, for the second test, I believe it would be better to ensure that all the keys are unique.

I think this is testing the same thing, but yeah your example is more concise, so I'll use that instead, thanks 👍